sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -27,17 +27,9 @@ export enum CloseReason {
 }
 
 /**
- * An interface representing the details on which instance of a `EventProcessor` owns processing
- * of a given partition from a consumer group of an Event Hub instance.
- *
- * **Note**: This is used internally by the `EventProcessor` and user never has to create it directly.
+ * An interface representing data that can identify a partition.
  */
-export interface PartitionOwnership {
-  /**
-   * @property The fully qualified Event Hubs namespace. This is likely to be similar to
-   * <yournamespace>.servicebus.windows.net
-   */
-  fullyQualifiedNamespace: string;
+export interface PartitionContext {
```

Comment:
fullyQualifiedNamespace may be useful as well so we can get at a truly unique set of values:
namespace/eventhub/consumerGroup/partitionId

Reply:
Done.